virt/virtual_machines/virt-vtpm-devices.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="virt-vtpm-devices_{context}"]
+= Virtual trusted platform module devices
```

Comment:
Please refer to my comment on the topic map file for advice regarding this title. (Make sure the topic map title and the title here match.)